Add bt coex jam (#6136)
nrfconnect/sdk-nrf: samples/bluetooth/radio_coex_1wire/README.rst

.. _bluetooth_radio_coex_1wire_sample:

Bluetooth: External radio coexistence using one-wire interface
################################################################

.. contents::
   :local:
   :depth: 2

> This sample demonstrates how to use the 1-Wire coexistence feature described in :ref:`Bluetooth® external radio coexistence <nrfxlib:bluetooth_coex>`.

Reviewer: Wire doesn't need to be capitalized here. 

Also, the title of the page uses "one-wire", and here is "1-wire". It's better to stick to using one or the other.

Overview
********

> The application sets up an advertiser and configures the 1wire coexistence feature.

Reviewer: ```suggestion
The application sets up an advertiser and configures the 1-wire coexistence feature.
```
Missing hyphen.

The user can enter ``g`` to grant or ``d`` to deny Bluetooth radio activity.

* When Bluetooth radio activity is granted, packets will be transmitted.

> * When Bluetooth radio activity is, no packets will be transmitted.

Reviewer 2: is denied

> * When Bluetooth radio activity is, no packets will be transmitted.

Reviewer: ```suggestion
* When Bluetooth radio activity is denied, no packets will be transmitted.
```
Missing word

* The number of packets transmitted is tracked by counting RADIO->READY event.
* The total number of transmitted packets will be printed every second.

The application generates a grant signal on the pin selected by ``coex-pta-grant-gpios``.
This pin has to be connected to the pin defined by the ``grant-gpios`` property in the DTS using a jumper cable.

* On ``nrf52840dk_nrf52840``, the default pins are **P0.26** and **P0.02**.

The board's :ref:`/zephyr,user <dt-zephyr-user>` node must have the ``coex-pta-grant-gpios`` property set in the device tree.
This sample's board overlay can be used as an example.

Requirements
************

The sample supports the following development kit:

.. table-from-rows:: /includes/sample_board_rows.txt
   :header: heading
   :rows: nrf52840dk_nrf52840

Building and running
********************

.. |sample path| replace:: :file:`samples/bluetooth/radio_coex_1wire`

.. include:: /includes/build_and_run.txt

Testing
=======

1. Build and program the development kit.
#. |connect_terminal|
#. Watch the number of packets transmitted.
#. Enter ``d`` to deny packet transmission.

> #. Observe that there are no packets transmitted, anymore.

Reviewer 2: no more packets transmitted. Or `Observe that packet transmissions have stopped`. And `Observe that packet transmission has re-started` or `started again`

> #. Observe that there are no packets transmitted, anymore.

Reviewer: ```suggestion
#. Observe that there are no packets transmitted anymore.
```

#. Enter ``g`` to grant packet transmission.

> #. Observe that packets are transmitted, again.

Reviewer: ```suggestion
#. Observe that packets are transmitted again.
```
